src/index.js

```diff
@@ -1,58 +1,63 @@
-'use strict'
-
-import _  from 'lodash'
-import fs  from 'fs'
-import config  from 'config'
-import Hapi  from 'hapi'
-import path  from 'path'
-import { default as pluginsConfig } from '../config/hapijs.plugins'
-import { default as log } from './logger'
+import _ from 'lodash';
+import fs from 'fs';
+import config from 'config';
+import Hapi from 'hapi';
+import path from 'path';
+import { default as pluginsConfig } from '../config/hapijs.plugins';
+import { default as log } from './logger';
 
 
 //
-// Global dependencies 
+// Global dependencies
 // (available across the whole App)
 //
-GLOBAL._ = _        // lodash
-GLOBAL.log = log    // Used instead of console()
+GLOBAL._ = _;        // lodash
+GLOBAL.log = log;    // Used instead of console()
 
 //
 // Create server
 //
-const server = new Hapi.Server()
-server.connection(config.get('server'))
+const server = new Hapi.Server();
+server.connection( config.get( 'server' ) );
+
+
+/**
+ * registerRoute - Register Routes to the server
+ *
+ * @param { Object } file
+ *
+ */
+const registerRoute = file => {
+  const routes = require( `./routes/${file}` ).default;
+  routes.forEach( route => server.route( route ) );
+};
 
 //
 // Register Hapi plugin's
 //
-server.register(pluginsConfig,
+server.register( pluginsConfig,
```

Comment:
Have a new way to register Plugins: https://github.com/cbmono/hapijs-seed-mvc/pull/57

You have to write a bit more though, but much more flexible. Specially when you have plugins that require more setup (ie. different Auth strategies)